<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArcFaceDetectionClass</class>
 <widget class="QMainWindow" name="ArcFaceDetectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ArcFaceDemo 3.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelPreview">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOperationCamera">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>495</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>启动摄像头</string>
    </property>
   </widget>
   <widget class="QListWidget" name="registerListWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>355</width>
      <height>480</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>495</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>注册人脸库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>495</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>选择识别照</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="editOutLog">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>550</y>
      <width>1000</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="cursorWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>495</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>清除人脸库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFaceCompare">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>495</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>人脸比对</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>505</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editThreshold">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOneRegister">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>495</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>注册单张人脸</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <action name="actionjihuo">
   <property name="text">
    <string>激活</string>
   </property>
  </action>
  <action name="actionpeizhi">
   <property name="text">
    <string>peizhi</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
